opencontext/server/routes/settings.py

```
@@ -177,6 +206,7 @@ async def update_model_settings(
         except HTTPException:
             raise
         except Exception as e:
+            logger.error(f"Failed to update model settings: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)